com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/HDShaderGUI.cs

```diff
@@ -63,8 +82,17 @@ public sealed override void OnGUI(MaterialEditor materialEditor, MaterialPropert
             }
         }
 
+        /// <summary>
+        /// Implement your custom GUI in this function.false You'll probably want to use the MaterialUIBlock to display a UI similar to HDRP shaders.
```

Comment:
Implement your custom GUI in this function. To display a UI similar to HDRP shaders, use the MaterialUIBlock.